packages/jaeger-ui/src/middlewares/fixtures/JaegerTrace.json

```diff
@@ -0,0 +1,344 @@
+{
```

Comment:
let's trim these fixtures to a single span. There's no value in multiple spans, the important part is really just the beginning of JSON, to ensure the right structure is used.

Reply:
I have implemented this and pushed the change, ive also change the fixture file names to be consistent with frontend